src/format/parsed.rs

```diff
@@ -166,6 +166,14 @@ impl Parsed {
         set_if_consistent(&mut self.year, value.to_i32().ok_or(OUT_OF_RANGE)?)
     }
 
+    /// Tries to set the [`year`](#structfield.year) field from given value.
+    #[inline]
+    pub fn set_year_short(&mut self, value: i64) -> ParseResult<()> {
```

Comment:
Is this used? I don't see it anywhere else in this PR. (possibly my search is broken)